av1an-core/src/target_quality.rs

```diff
@@ -191,46 +190,48 @@ impl TargetQuality {
                 tokio::process::Command::new(pipe_cmd)
                     .args(args)
                     .stderr(if cfg!(windows) {
-                        Stdio::null()
+                        std::process::Stdio::null()
                     } else {
-                        Stdio::piped()
+                        std::process::Stdio::piped()
                     })
-                    .stdout(Stdio::piped())
+                    .stdout(std::process::Stdio::piped())
                     .spawn()
                     .unwrap()
             } else {
                 unreachable!()
             };
 
-            let source_pipe_stdout: Stdio = source.stdout.take().unwrap().try_into().unwrap();
+            let source_pipe_stdout: std::process::Stdio =
+                source.stdout.take().unwrap().try_into().unwrap();
 
             let mut source_pipe = if let [ffmpeg, args @ ..] = &*cmd.0 {
                 tokio::process::Command::new(ffmpeg)
                     .args(args)
                     .stdin(source_pipe_stdout)
-                    .stdout(Stdio::piped())
+                    .stdout(std::process::Stdio::piped())
```

Comment:
Was there an issue that led to needing to use std's Stdio instead of tokio's here?